apps/qubit/modules/digitalobject/actions/metadataComponent.class.php

```diff
@@ -36,7 +36,8 @@ public function execute($request)
             && arStorageServiceUtils::getAipDownloadEnabled()
         );
 
-        $this->masterCopy = $this->resource->getRepresentationByUsage(QubitTerm::MASTER_ID);
+        $id = QubitTerm::MASTER_ID;
+        $this->masterCopy = $this->resource->id;
```

Comment:
Declaring a `$this->masterCopy` variable doesn't seem necessary, as it's only used once (line 208), and the assigned value can be tested directly, e.g.

```
        ) && null !== $this->masterCopy;
```

Can be changed to:

```
        ) && isset($this->resource->id);
```

I believe using `isset()` instead of `null !== $this->resource->id` avoids a raising a warning if `$this->resource` is null.